packages/x-date-pickers/src/internals/models/props/basePickerProps.tsx

```diff
@@ -66,3 +66,7 @@ export interface BaseNextPickerProps<TValue, TDate, TView extends CalendarOrCloc
    */
   localeText?: PickersInputComponentLocaleText<TDate>;
 }
+
+/**
+ * Props common to all non-static pickers
+ */
```

Comment:
Should it be moved on top of the `UsePickerValueNonStaticProps`?

Reply:
:laughing: Oh I forgot to remove this comment
I was going to create a `BaseNextNonStaticPickerProps` but in the end I went for the `MobileOnly` and `DesktopOnly` interfaces.
I'll remove this comment

Reply:
I can add a description there indeed